### ISSUE TITLE
Gives device not found on C128 in C128 mode.

### ISSUE THREAD

**Issue by abcbarryn** [Ghouli/sd2iec#1]
C64 mode works fine. C128 mode leaves the red LED on at first disk access and displays device not found. I attached the config file for my hardware.

[config-sw2-m644.txt](https://github.com/Ghouli/sd2iec/files/1990553/config-sw2-m644.txt)

**Comment by Ghouli** [comment]
Yeah, the fast serial patch isn't currently working and requires a fix. It works fine in C64 mode, but in C128 mode this tries to use burst load capabilities, which are currently broken. Someone should probably look into that when they have time...

**Comment by abcbarryn** [comment]
I thought fast serial only worked in C128 mode anyway? Is there an easy way to turn off/disable burst mode in the code until it can be fixed?

**Comment by abcbarryn** [comment]
Still, 2 years later, hoping someone will find a way to make this work. :(

**Comment by abcbarryn** [comment]
Interesting note:
LOAD"$",8
LIST
Actually works in C128 mode, but
LOAD"*",8
Does not.
